src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue

```diff
@@ -44,6 +47,36 @@ export default {
   },
   setup() {
     return {
+      question1: {
+        Name: "Instruction",
+        Text: "How much do you value instructions from your teachers?",
+        Min: 1,
+        MinLabel: "I'd like more time studying alone.",
+        Max: 7,
+        MaxLabel: "I wish instructors can cover every details",
+      },
+      question2: {
+        Name: "Supportive Environment",
+        Text: "How much do you weight the importance of a supportive environment?",
+        Min: 1,
+        MinLabel: "I'm a loner for life.",
+        Max: 7,
+        MaxLabel: "I want a family like school environment!",
+      },
+      question3: {
+        Name: "Borough",
+        Text: "Is there a spicific borough you are looking for?",
```

Comment:
```suggestion
        Text: "Is there a specific borough you are looking for?",
```

Reply:
Same goes for the other question locations

Reply:
fixed, all three